pkg/converter/convert_unix.go

```diff
@@ -621,24 +626,32 @@ func ConvertHookFunc(opt MergeOption) converter.ConvertHookFunc {
 			// blob layer into manifest if blob storage backend is specified.
 			manifest.Layers = []ocispec.Descriptor{*bootstrapDesc}
 		} else {
-			manifest.Layers = append(manifest.Layers, *bootstrapDesc)
+			for idx, blobDesc := range blobDescs {
+				blobGCLabelKey := fmt.Sprintf("containerd.io/gc.ref.content.l.%d", idx)
+				labels[blobGCLabelKey] = blobDesc.Digest.String()
+			}
+			// Affected by chunk dict, the blob list referenced by final bootstrap
+			// are from different layers, part of them are from original layers, part
+			// from chunk dict bootstrap, so we need to rewrite manifest's layers here.
```

Comment:
I think overwrite is better than rewrite

Reply:
It means just generating a new manifest, not covering it. :)